### ISSUE TITLE
Can't open CSV files

### ISSUE THREAD

**Issue by 0x07xxs** [openaddresses/openaddresses#7489]
Currently I want to download the CSV data for the San Diego county but for some reason on the openaddressio website doesn't let me when I click to download the CSV data for the addresses or parcels. I'm not sure what I'm doing wrong.

![image](https://github.com/user-attachments/assets/033a3d4f-5165-4c2d-9c08-bee8e9c7148d)

**Comment by iandees** [comment]
Is there an error dialog that shows up or anything in your browser developer console?

**Comment by 0x07xxs** [comment]
> Is there an error dialog that shows up or anything in your browser developer console?

No, it doesn't say anything I tried going on the same website on a different browser to download it I used chrome in this instance and it was the same issue. If there is an alternative version to install this using commands I would appreciate it but in the meantime I right click where it says "CSV" and there's not even a redirect link to open on a new tab or anything.

**Comment by michaelblyons** [comment]
Same for me on Firefox.

**Edit:** I can access the GeoJSON. Perhaps there should be a message that one's tier doesn't support CSV access when one clicks it?

**Comment by iandees** [comment]
Based on my investigation: The "CSV" text in the download arrow always kicks off an export task, even if one is in progress for the selected data or if it was recently completed. Sometimes that process of kicking off an export can take a while, and there's no indication that any work is happening so it appears like the click does nothing.

What's supposed to happen is that it redirects you to the export task in progress so you can watch it and then click the download link for the exported data.

We should probably add some kind of spinner that indicates some action is being taken and disable the link so a user doesn't click it multiple times.

In the mean time, head to https://batch.openaddresses.io/export and download your exported data from there.
